test/physics/em/UrbanMsc.test.cc

```diff
@@ -214,25 +214,27 @@ TEST_F(UrbanMscTest, msc_scattering)
     MscInteraction sample_result;
 
     // Input
-    const unsigned int nsamples = 8;
-
-    real_type energy[nsamples] = {51.0231,
-                                  10.0564,
-                                  5.05808,
-                                  1.01162,
-                                  0.501328,
-                                  0.102364,
-                                  0.0465336,
-                                  0.00708839};
-
-    real_type step[nsamples] = {0.00279169,
-                                0.412343,
-                                0.0376414,
-                                0.178529,
-                                0.0836231,
-                                0.125696,
-                                0.00143809,
-                                0.105187};
+    static const real_type energy[] = {51.0231,
+                                       10.0564,
+                                       5.05808,
+                                       1.01162,
+                                       0.501328,
+                                       0.102364,
+                                       0.0465336,
+                                       0.00708839};
+
+    static const real_type step[] = {0.00279169,
+                                     0.412343,
+                                     0.0376414,
+                                     0.078163296576415602,
```

Comment:
Why these input values are changed?

Reply:
I changed them because the longer steps violated the new range limits: the values are now equal to the energy loss range, as would be encountered during a real problem.